### ISSUE TITLE
Weather widget covered with "Check your settings" pop-up

### ISSUE THREAD

**Issue by tomasz1986** [valinet/ExplorerPatcher#2767]
This seems to happen almost everyday. After clearing weather widget local data, the weather seems to work for a few hours, but then the pop-up comes back.

![image](https://github.com/valinet/ExplorerPatcher/assets/5626656/a82598fc-435c-464c-b446-b36dafb85061)

Windows 10 Enterprise LTSC 2021, ExplorerPatcher v22621.2861.62.2.

**Comment by pyrates999** [comment]
Please post windows build.  You can find this by running winver.

**Comment by tomasz1986** [comment]
Windows 10 Enterprise LTSC 2021 is equivalent to Windows 10 version 21H2 (OS Build 19044.3930).

**Comment by nexusV2** [comment]
Same here Win 11 (22631.3007)

**Comment by pyrates999** [comment]
The weather results are coming from google.  And that popup is caused by google.  So I don't think there's anything that EP can do about that.

**Comment by tomasz1986** [comment]
This looks like the same problem as before in https://github.com/valinet/ExplorerPatcher/issues/934. Basically, the CSS class/id has likely changed (as its randomly generated), so EP would need to block the current one (or use a different approach to block the element without relying on the class/id, which will surely change again in the future).

**Comment by tomasz1986** [comment]
> #934 is the exact same issue. In that case, you must resize the window and set the setting required.

Yeah, the problem is that this is a game that has no end, because the current CSS class name, being randomly generated, is going to change again in the future. It would be nice to have a more permanent solution, e.g. by targeting the HTML pop-up elements depending on their position on the site.

**Comment by pyrates999** [comment]
You'll just have to wait and see then.

**Comment by realemperorart** [comment]
I have the same problem and I also can't log in to google because when I click the button the "Unable to load weather information" message shows.

<img width="509" alt="Screenshot 2024-02-07 150718" src="https://github.com/valinet/ExplorerPatcher/assets/159272816/921f9634-394a-409f-984b-6e6ebb38dcba">

**Comment by pyrates999** [comment]
This can't be fixed because it's coming straight from google's website.

**Comment by realemperorart** [comment]
Even if i could log in there?

**Comment by pyrates999** [comment]
You can try to login.  But I don't think it saves cookies.  So it will ask you again next time that url is accessed.

**Comment by realemperorart** [comment]
Well i cant even log in because the window is too small and it gives me the "Unable to load weather information" as soon as a click the button to go to the login screen anyways... so you say its just not really usable anymore?

**Comment by pyrates999** [comment]
Not if google is asking you to login to get the weather.

**Comment by nexusV2** [comment]
Was fixed a few days after Update but since today it´s covered again. ^^

**Comment by pyrates999** [comment]
This can't be fixed by EP.  It's caused by google.  Sometimes when I'm searching for something and I'm not signed in, it will prompt me to sign in.  I just dismiss it every time.

**Comment by nexusV2** [comment]
I am generally logged in with my Google account but I didn´t find a special login for Google weather service.

**Comment by pyrates999** [comment]
The weather feature in EP doesn't use your google login.

**Comment by dkwarr87** [comment]
I've been having the same issue for a few months now. I was able to clear it today by enabling [Weather Debug Mode](https://github.com/valinet/ExplorerPatcher/wiki/Weather#i-am-facing-a-layout-or-some-other-issue-with-the-content-displayed-by-the-weather-widget-is-there-a-way-to-debug-it) which let me expand the window and click on the "Got it" button, then disabling/re-enabling the weather widget. I'll see how long that lasts.
